src/SourceBuild/tarball/content/eng/bootstrap/README.template.md

```diff
@@ -0,0 +1,159 @@
+See also [dotnet/source-build](https://github.com/dotnet/source-build) for more information about our whole-product source-build.
+
+## Goals
+
+- The main purpose of the [dotnet/dotnet](https://github.com/dotnet/dotnet) repository is to have all source code necessary to build the .NET product available in one repository and identified by a single commit.
+- The VMR also aims to become the place from which we release and service future versions of .NET to reduce the complexity of the product construction process. This should allow our partners and and 3rd parties to easily build, test and modify .NET using their custom infrastructure as well as make the process available to the community.
+- Lastly, we hope to solve other problems that the current multi-repo setup brings:
+    - Enable the standard [down-/up-stream open-source model](https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild/VMR-Upstream-Downstream.md).
+    - Fulfill requirements of .NET distro builders such as RedHat or Canonical to natively include .NET in their distribution repositories.
+    - Simplify scenarios such as client-run testing of bug fixes and improvements. The build should work in an offline environment too for certain platforms.
+    - Enable developers to make and test changes spanning multiple repositories.
+    - More efficient pipeline for security fixes during the CVE pre-disclosure process.
+
+## Limitations
+
+**This is a work-in-progress and an experiment.**
+There are considerable limitations to what is possible at the moment. For an extensive list of current limitations, please see [Temporary Mechanics](https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild/VMR-Design-And-Operation.md#temporary-mechanics).
+
+The VMR is expected to become non-experimental by .NET 8 Preview 1 (Februrary, 2023) 
+This means it won't be short-lived anymore and we won't be reserving the right to delete and re-create it anymore.
+Other limitations might apply until the .NET 9 timeframe.
+See the [Unified Build roadmap](https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild/Roadmap.md) for more details.
+
+### Supported platforms
+
+The VMR only supports .NET 8.0 and higher. Additionally, source-build currently supports Linux only.  
+It is expected that Mac and Window will be supported in the .NET 9.0.
+
+For the latest information about Source-Build support for new .NET versions, please check our [GitHub Discussions page](https://github.com/dotnet/source-build/discussions) for announcements.
+
+### Online build only
+
+Building the product offline is not fully working at the moment. The `--online` switch is needed when building the VMR as not all dependencies are currently built from source.
+
+### Code flow
+For the time being, the source code only flows one way - from the individual repos into the VMR.
+Changes done to the VMR are not automatically mirrored back. More details on this process:
```

Comment:
This implies changes are allowed.   This is clarified later on.  Thoughts on clarifying this here?

Reply:
Good idea, let's be as explicit as possible. Redundancy doesn't hurt imho